main.go

```diff
@@ -226,6 +228,13 @@ func InitFlags(fs *pflag.FlagSet) {
 	fs.IntVar(&machineHealthCheckConcurrency, "machinehealthcheck-concurrency", 10,
 		"Number of machine health checks to process simultaneously")
 
+	fs.StringArrayVar(&machineSetPreflightChecks, "machineset-preflight-checks", []string{
+		string(clusterv1.MachineSetPreflightCheckAll)},
```

Comment:
Wondering if that should be instead the full comma separated list (`KubeadmVersionSkew,KubernetesVersionSkew,ControlPlaneIsStable,ControlPlaneVersionSkew` instead :think:

Reply:
I don't have a strong opinion. All just seemed shorter and less prone to mistakes if we add an additional check and forget to change the value here